flight0handler.go

```diff
@@ -6,14 +6,12 @@ import (
 )
 
 func flight0Parse(ctx context.Context, c flightConn, state *State, cache *handshakeCache, cfg *handshakeConfig) (flightVal, *alert, error) {
-	// HelloVerifyRequest can be skipped by the server,
-	// so allow ServerHello during flight1 also
-	seq, msgs, ok := cache.fullPullMap(state.handshakeRecvSequence,
+	seq, msgs, ok := cache.fullPullMap(0,
```

Comment:
cache must have a flag indicates the message was already processed.
otherwise, any non-ClientHello packet may trigger HelloVerifyRequest.

Reply:
Only handshake packet can triggering it.
One response per one packet is OK in point of DDoS prevention.